### DISCUSSION TITLE
如何让机器人不跟随系统启动

### DISCUSSION THREAD

**Discussion by Seiryuu-Qiu** [simple-robot/Discussions#51] [Q&A]
在applicaition.yml中设置了 auto-start-bots: false
![image](https://user-images.githubusercontent.com/35923395/209920117-a4119f5c-2a1d-4eee-b3c6-eabd13166f7c.png)
启动之后 还是跟随系统启动了
![image](https://user-images.githubusercontent.com/35923395/209920302-3be0d4ec-73de-4544-b22b-849d9d7ad149.png)

**Comment by ForteScarlet** [comment]
尝试更新核心版本，较早版本存在配置不生效的问题，可能与 https://github.com/simple-robot/simpler-robot/issues/550 是类似问题
